src/installation/guides/arm-devices/index.md

```diff
@@ -0,0 +1,137 @@
+section is only necessary for [tarball](#tarball-installation) and
+[chroot](#chroot-installation) installations.
+
+The usual partitioning scheme for ARM devices requires at least two partitions
+on a drive formatted with an MSDOS partition table:
+
+- one formatted as FAT32 with partition type `0c`, which will be mounted on
+   `/boot`;
+- one that can be formatted as any file system that Linux can boot from, such as
+   ext4, which will be mounted on `/`. If you are using an SD card, for example,
+   it might be interesting to create the ext4 file system with the
+   `^has_journal` option, which disables journaling and might increase the
+   drive's life, at the cost of a higher chance of data loss.
+
+This can be done with the [cfdisk(8)](https://man.voidlinux.org/cfdisk.8)
+utility, for example.
+
+To access the newly created file systems, it is necessary to mount them. This
+guide will assume that the second partition will be mounted on `/mnt`, but you
+may mount it elsewhere. To mount these filesystems, you can use the commands
+below, replacing the device names with the appropriate ones for your setup.
+
+```
+# mnt /dev/mmcblk0p2 /mnt
+# mkdir /mnt/boot
+# mount /dev/mmcblk0p1 /mnt/boot
+```
+
+### Tarball installation
+
+First, [download and verify](../../index.md#downloading-installation-media) a
+PLATFORMFS or ROOTFS tarball for your desired platform and [prepare your storage
+medium](#preparing-your-storage). Then, it is necessary to unpack the tarball
+onto the file system, which can be done using
+[tar(1)](https://man.voidlinux.org/tar.1):
+
+```
+# tar xvfp <image>.tar.xz -C /mnt
+```
+
+### Chroot installation
+
+It is also possible to perform a chroot installation, which requires using the
+`qemu-user-static` package together with either the `binfmt-support` or `proot`
+package. This guide explains how to use the `qemu-<platform>-static` program
+from `qemu-user-static` with [proot(1)](https://man.voidlinux.org/proot.1).
+
+First, [download and verify](../../index.md#downloading-installation-media) a
+PLATFORMFS or ROOTFS tarball for your desired platform and [prepare your storage
+medium](#preparing-your-storage). Then, follow the [XBPS chroot installation
+steps](../chroot.md#the-xbps-method) using the appropriate architecture and base
+packages, some of which are listed in the "[Supported
+platforms](./platforms.md)" page.
+
+Finally, follow the [chroot configuration steps](../chroot.md#configuration)
+steps, but instead of using the [chroot(1)](https://man.voidlinux.org/chroot.1)
+command to [enter the chroot](../chroot.md#entering-the-chroot), use the command
+below:
+
+```
+proot -q qemu-arm-static -r /mnt -w /
+```
+
+## Configuration
+
+Some additional configuration steps need to be followed to guarantee a working
+system. Configuring a [graphical
+session](../../../config/graphical-session/index.md) should work as normal.
+
+### Logging in
+
+The `root` password is `voidlinux`.
```

Comment:
This is only true for the prebuilt images as well as the tarballs. not for installation via chroot where you set it yourself

Reply:
Oh good point, thanks.

Reply:
Fixed, I think.